Examples/iOSExplorer/Podfile.lock

```diff
@@ -0,0 +1,33 @@
+PODS:
+  - ADALiOS (1.2.4)
+  - OneDriveSDK (1.1.2):
+    - OneDriveSDK/Auth (= 1.1.2)
+    - OneDriveSDK/Common (= 1.1.2)
+    - OneDriveSDK/Extensions (= 1.1.2)
+    - OneDriveSDK/Implementations (= 1.1.2)
+    - OneDriveSDK/OneDriveCoreSDK (= 1.1.2)
+  - OneDriveSDK/Auth (1.1.2):
+    - ADALiOS (~> 1.2.4)
+    - OneDriveSDK/Common
+  - OneDriveSDK/Common (1.1.2)
+  - OneDriveSDK/Extensions (1.1.2):
+    - OneDriveSDK/Auth
+    - OneDriveSDK/Implementations
+    - OneDriveSDK/OneDriveCoreSDK
+  - OneDriveSDK/Implementations (1.1.2):
+    - OneDriveSDK/Common
+  - OneDriveSDK/OneDriveCoreSDK (1.1.2):
+    - OneDriveSDK/Common
+
+DEPENDENCIES:
+  - OneDriveSDK (from `../../`)
```

Comment:
It's probably safer having Podfile.lock in there (in fact its purpose is to ensure a consistent view of the world by multiple developers working on the same project) if we want the sample app to 'just work' when cloned. Alternately, I could see there being some argument for removing it since keeping it updated to latest versions of pods is a maintenance cost.

If we want to get rid of it, we should just add Podfile.lock to the gitignore. That should be a safe enough change, as we want the same behavior in both the SDK and the sample app, and projects cloning the SDK as a submodule or cocoapod won't be affected by it.